spec/support/request_helpers.rb

```diff
@@ -0,0 +1,5 @@
+module RequestHelpers
+  def json_response
+    @json_response ||= JSON.parse(response.body)
```

Comment:
May be more prudent to use `@_json_response` for a memoised value here.

Reply:
@misaka Currently this raises a rubocop violation https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Naming/MemoizedInstanceVariableName. We would need to change `EnforcedStyleForLeadingUnderscores: disallowed (default)` to something like `required` or `optional`, thoughts?

Reply:
As discussed, I'd be happy with `optional` here. I personally prefer the underscore here, but I understand that it's not always required.

Reply:
done https://github.com/DFE-Digital/teacher-training-api/pull/1528/commits/991061d39e4caf7deaf073f8e1f29c929a2a089a